src/components/ServiceList.jsx

```diff
@@ -1,22 +1,67 @@
 import { Alert } from "@baltimorecounty/dotgov-components";
 import FilterList from "../components/FilterList";
-import { MostPopularServiceIconStyles } from "../styles";
-import React from "react";
-import ServiceIconLink from "../components/ServiceIconLink";
+import React, { useState } from "react";
+import ServiceIconLink from "./ServiceIconLink";
+import SearchCollapse from "./SearchCollapse";
 import useServices from "../hooks/useServices";
+import {
+  MostPopularServiceIconStyles,
+  LegendText,
+  CollapseContainer
```

Comment:
i think this is a deepscan error.

Reply:
yeah that wasnt being used but was imported. Its all been removed